pandas/types/validate.py

```diff
@@ -0,0 +1,7 @@
+from pandas.types.common import is_bool
+
+def _validate_bool_type(value, arg_name):
+    if not (is_bool(value) or value is None):
+        raise ValueError('For argument %s expected type bool, received type %s.' %\
```

Comment:
Can you insert quotes around `%s`? (make it clearer that is the arg name), like `"Argument '%s' exp ..."`